code/game/gamemodes/clock_cult/clock_scriptures/scripture_drivers.dm

```diff
@@ -129,7 +129,7 @@
 
 /datum/clockwork_scripture/abscond/scripture_effects()
 	var/take_pulling = invoker.pulling && isliving(invoker.pulling) && get_clockwork_power(ABSCOND_ABDUCTION_COST)
-	var/turf/T = get_turf(pick(GLOB.servant_spawns))
+	var/turf/T = get_step(GLOB.ark_of_the_clockwork_justiciar, SOUTH)
```

Comment:
needs a sanity check just in case they're trying to return after the ark got blown up